api/src/main/java/io/grpc/ServerBuilder.java

```diff
@@ -89,6 +90,16 @@
    */
   public abstract T addService(BindableService bindableService);
 
+  /**
+   * Adds a list of service implementations to the handler registry together.
+   *
+   * @param bindableServices the list of BindableService objects
+   * @return this
+   * @since 1.37.0
+   */
+  @ExperimentalApi("https://github.com/grpc/grpc-java/issues/7925")
+  public abstract T addServices(List<BindableService> bindableServices);
```

Comment:
I was thinking we'd more likely want `List<ServerServiceDefinition>`, since that is the lowest-level form and allows you to even wrap the services with interceptors if you need to. I'm not against this argument, but sort of expected there wouldn't be much need or it.

Reply:
What about keeping both?

Reply:
Well, nvm. We cannot do both...

Reply:
Oh... generics. I didn't think of that issue. If we can only have one then ServerServiceDefinition is the winner. We could use arrays instead of lists, although that sort of kills the existing addService since `.addService(blah).addService(blah2)` would just become `.addService(blah, blah2)` most of the time. (Might not be bad, but it has more consequences.)

As I said, I don't think there's all that much value of `List<ServerServiceDefinition>`, although I do think there'd be some limited value in `BindableService...` if we went with that approach. But I figure "nah, just do what we need; we can always add more later but it is hard to take something away."

Reply:
In either case this will just loop thru the list and call the corresponding `addService(...)`. Also it will be useful to have both- `List<ServerServiceDefinition>` or `List<BindableService>` depending on what the caller has